src/pages/news-announcements.tsx

```diff
@@ -0,0 +1,115 @@
+import { InferGetServerSidePropsType } from 'next'
+import React, { useEffect } from 'react'
+import {
+  BreadcrumbBar,
+  Breadcrumb,
+  BreadcrumbLink,
+  Grid,
+} from '@trussworks/react-uswds'
+import { client } from '../lib/keystoneClient'
+import { withPageLayout } from 'layout/DefaultLayout/PageLayout'
+import Announcement from 'components/Announcement/Announcement'
+import Loader from 'components/Loader/Loader'
+import LoadingWidget from 'components/LoadingWidget/LoadingWidget'
+import { useUser } from 'hooks/useUser'
+import LinkTo from 'components/util/LinkTo/LinkTo'
+import NavLink, { NavLinkProps } from 'components/util/NavLink/NavLink'
+import { useRSSFeed } from 'hooks/useRSSFeed'
+import styles from 'styles/pages/news.module.scss'
+import type { RSSNewsItem } from 'types'
+import { validateNewsItems, formatToArticleListItem } from 'helpers/index'
+import { GET_ANNOUNCEMENTS } from 'operations/cms/queries/getAnnouncements'
+import { SPACEFORCE_NEWS_RSS_URL } from 'constants/index'
+import { ArticleList } from 'components/ArticleList/ArticleList'
+
+const RSS_URL = `${SPACEFORCE_NEWS_RSS_URL}&max=12`
+
+const NewsAnnouncements = ({
```

Comment:
Question: This PR is dependent on PR #723 correct?

Reply:
Correct. I forgot to mention that in this PR. I'll be sure to merge that one first.